File: Projekt1_cpp/Projekt1_cpp/Trawa.cpp
#include "Trawa.h"

Trawa::Trawa(pair<int, int>pole, Swiat* swiat)
	:Roslina(0, 0, pole, swiat) {}

Organizm* Trawa::potomek(pair<int, int>pole) {
	return new Trawa(pole, swiat);
}

void Trawa::rysowanie() {
	cout << 't';
}

string Trawa::getNazwa() {
	return "trawa";
}

File: Projekt2_java/src/projekt2/rosliny/Trawa.java
package projekt2.rosliny;

import projekt2.*;

import java.awt.*;

public class Trawa extends Roslina {

    public Trawa(Pair pole, Swiat swiat) {
        super(0, 0, pole, swiat);
    }

    @Override
    public Organizm potomek(Pair pole) {
        return new Trawa(pole, swiat);
    }

    @Override
    public Color rysuj() {
        return Color.GREEN.brighter();
    }

    @Override
    public String getNazwa() {
        return "trawa";
    }
}
